### DISCUSSION TITLE
pronouns.cc

### DISCUSSION THREAD

**Discussion by aquifolly** [catppuccin/catppuccin#2484] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://pronouns.cc
a userstyle

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by NeonGamerBot-QK** [comment]
![](https://cdn.discordapp.com/emojis/986539896409841684.png?v=1&size=16) https://github.com/catppuccin/userstyles/pull/1140 made a PR here

**Comment by sgoudham** [comment]
This userstyle now exists
